src/otx/core/data/transform_libs/torchvision.py

```diff
@@ -368,6 +369,7 @@ def __init__(
         min_ious: Sequence[float] = (0.1, 0.3, 0.5, 0.7, 0.9),
         min_crop_size: float = 0.3,
         bbox_clip_border: bool = True,
+        p: float = 1.0,
```

Comment:
Could you update docstring for this argument?

Reply:
Done https://github.com/openvinotoolkit/training_extensions/pull/3741/commits/28837861e6f6151bef8608a940b020bc6ed22437